server/middleware/setUpWebSecurity.ts

```diff
@@ -18,33 +18,24 @@ export default function setUpWebSecurity(): Router {
     helmet({
       contentSecurityPolicy: {
         directives: {
-          defaultSrc: ["'self'"],
-          formAction: ["'self'", new URL(config.apis.hmppsAuth.url).hostname],
-          scriptSrc: [
+          defaultSrc: [
```

Comment:
diff on GH is a bit confusing, I've removed existing hotjar entries and moved everything in `defaultSrc`, rest should be as it was before trying to add hotjar.